### PR TITLE
Gamepad response curve cosmetic changes

### PR DESCRIPTION
**Pull request by ceski-1**

### CODE REVIEW

File: src/m_menu.c [hunk 1]
```diff
@@ -4185,6 +4185,7 @@ static const char *layout_strings[] = {
 };
 
 static const char *curve_strings[] = {
+  "", "", "", "", "", "", "", "", "", "", // Dummy values, start at 1.0.
```
Comment (ceski-1) [inline src/m_menu.c:4188]: I wonder if there's a better way to do this.

Comment (rfomin) [reply src/m_menu.c:4188]: This solution is alright.

This solution is fine.

It's not that important what indexes are stored in config. For most users, `(10 = Linear, 20 = Squared, 30 = Cubed)` is as cryptic as `(0 = Linear, 10 = Squared, 20 = Cubed)`. Printing and reading floats is challenging (see https://netlib.org/fp/dtoa.c) and, as @fabiangreffrath points out, there are still cross-platform issues. I don't think we need sophisticated UI widgets with float support for the Doom menu.

**Review by rfomin — APPROVED** [review]

File: src/m_menu.c (hunk 1 above)
Comment (ceski-1) [reply src/m_menu.c:4188]: Sounds good to me. Should we move forward with these cosmetic changes? "10/20/30" is slightly more intuitive for me, and should make looking at this code a little less cryptic in the future. It's not critical though.

Comment (rfomin) [reply src/m_menu.c:4188]: > Should we move forward with these cosmetic changes?

Sure. If it doesn't affect user configs from the previous version, I'm fine with any cosmetic change.

**Pull request merged**